controllers/api/v1/keyword.go

```diff
@@ -20,14 +24,56 @@ func (c *KeywordController) NestPrepare() {
 
 // URLMapping maps keyword controller actions to functions
 func (c *KeywordController) URLMapping() {
+	c.Mapping("Index", c.Index)
 	c.Mapping("TextSearch", c.TextSearch)
 }
 
 // actionPolicyMapping maps keyword controller actions to policies
 func (c *KeywordController) actionPolicyMapping() {
+	c.MappingPolicy("Index", Policy{RequireAuthenticatedUser: true})
 	c.MappingPolicy("TextSearch", Policy{RequireAuthenticatedUser: true})
 }
 
+// Index handles keyword list
+// @Title Index
+// @Description response with keyword list filterable with keyword and page number
+// @Success 200 {object} v1serializers.KeywordList
+// @Param keyword	query string	false
+// @Param p			query integer	false
+// @Failure 500 Internal Server Error
+// @Accept json
+// @router /api/v1/keywords [get]
+func (c *KeywordController) Index() {
+	keyword := c.GetString("keyword")
+	queryList := map[string]interface{}{
+		"user_id":            c.CurrentUser.Id,
+		"keyword__icontains": keyword,
+	}
+
+	totalRows, err := models.CountAllKeyword(queryList)
+	if err != nil {
+		c.RenderGenericError(ErrorRetrieveKeywordFailed)
+
+		return
```

Comment:
I think we can remove the `return` here as we're using `StopRun()` in `renderError()`, right?